.travis.yml

```diff
@@ -40,58 +41,68 @@ matrix:
 
         # Try MacOS X
         - os: osx
-          env: PYTHON_VERSION=2.7 SETUP_CMD='test' OPTIONAL_DEPS=true
+          env: PYTHON_VERSION=2.7 SETUP_CMD='test'
 
         # Check for sphinx doc build warnings - we do this first because it
-        # runs for a long time
+        # runs for a long time. WCSAxes is a dependency for the docs.
         - os: linux
-          env: PYTHON_VERSION=2.7 SETUP_CMD='build_sphinx -w'  OPTIONAL_DEPS=true
-          # OPTIONAL_DEPS needed because the plot_directive in sphinx needs them
+          env: PYTHON_VERSION=2.7 SETUP_CMD='build_sphinx -w'
+               PIP_DEPENDENCIES='beautifulsoup4 wcsaxes'
 
-        # Try all python versions with the latest numpy
+        # Try all python versions with the latest numpy, but without the
+        # optional dependencies
         - os: linux
           env: PYTHON_VERSION=2.6 SETUP_CMD='test --open-files'
+               CONDA_DEPENDENCIES='Cython jinja2' PIP_DEPENDENCIES=''
         - os: linux
           env: PYTHON_VERSION=2.7 SETUP_CMD='test --open-files'
+               CONDA_DEPENDENCIES='Cython jinja2' PIP_DEPENDENCIES=''
         - os: linux
           env: PYTHON_VERSION=3.3 SETUP_CMD='test --open-files'
+               CONDA_DEPENDENCIES='Cython jinja2' PIP_DEPENDENCIES=''
```

Comment:
I think it maybe makes more sense to define these (``CONDA_DEPENDENCIES='Cython jinja2' PIP_DEPENDENCIES=''``) as the global defaults since they are the most common and to specify the dependencies for the optional dependency builds?